otomoto/settings.py

```diff
@@ -70,14 +74,14 @@
 
 # Enable and configure the AutoThrottle extension (disabled by default)
 # See https://doc.scrapy.org/en/latest/topics/autothrottle.html
-#AUTOTHROTTLE_ENABLED = True
+AUTOTHROTTLE_ENABLED = True
```

Comment:
Why we need throttling?

Reply:
In documentation i read that is nicer to sites instead of using default download delay of zero so I thought it would be good to change it

Reply:
Yes, but due to amount of cars I think it's better to speed things up.  Did you manage to run it fully?  I did once and I was not banned.

Reply:
Yes I run it fully. I was not banned. I comment this setting